back-up-and-restore-data.md

```diff
@@ -105,11 +106,14 @@ INSERT INTO episodes (id, season, num, title, stardate) VALUES
 ### Dump just a table's schema
 
 ~~~ shell
-$ cockroach dump startrek episodes --user=maxroach --dump-mode=schema > backup.sql
+$ cockroach dump startrek episodes --insecure --user=maxroach --dump-mode=schema > backup.sql
 ~~~
 
 ~~~ shell
 $ cat backup.sql
+~~~
```

Comment:
Haha, we've started splitting command and output into different boxes to improve clarity. Trying to be consistent here. When we revisit docs style issues, I'll make sure this is a part of the review.